docs(oidc): add peertube

0xSysR3ll opened the pull request:
This adds peertube to the OpenID Connect 1.0 integration docs.

james-d-elliott commented on docs/content/integration/openid-connect/peertube/index.md:
```diff
@@ -0,0 +1,110 @@
+   2. Visit `Plugins/Themes`.
+   3. Visit `Installed plugins`.
+   4. Click the `Settings` button of the installed plugin.
+   5. Enter the following configuration :
```
```suggestion
   5. Enter the following configuration:
```

james-d-elliott reviewed (changes requested):
Mostly LGTM. Could you also fix this one issue and provide a screenshot if this is a GUI? Use the domain `peertube.example.com` and `auth.example.com`, as well as the secret `insecure_secret`. Also is there no option on their end to configure the client id?

0xSysR3ll commented:
Hello,
Thanks for the review.
I fixed the issues reported, and added the missing `Client ID` parameter that is indeed asked by the PeerTube plugin (see the screenshot below).

![PeerTube_UI](https://github.com/user-attachments/assets/f0ce1766-fd51-426d-b6b7-077980e40a3a)

0xSysR3ll reviewed (commented):
Hello,
Thanks for the review.
I fixed the issues reported, and added the missing Client ID parameter that is indeed asked by the PeerTube plugin (see the screenshots below).
I'm not sure if that's what you asked for.

![PeerTube_UI_1](https://github.com/user-attachments/assets/dc251467-a2a5-4a44-9004-c99249edb2b1)
![PeerTube_UI_2](https://github.com/user-attachments/assets/f1f3aa38-484b-49bc-9947-9e836c4a8e07)

james-d-elliott commented:
Thanks a lot for choosing to contribute. @all-contributors please add @0xSysR3ll for docs.
